model_analyzer/config/input/config_command_profile.py

```diff
@@ -596,6 +596,15 @@ def _add_run_search_configs(self):
                 description=
                 "Value for the model's max_batch_size that run config search will start from."
             ))
+        self._add_config(
+            ConfigField(
+                'run_config_search_max_bcs_steps',
```

Comment:
Is another story going to tie these two stories together? Or are you just waiting for the first one to merge before you start using the new option?

Reply:
Just planned to make this change at the branch level